core/git_command.py

```diff
@@ -96,6 +96,11 @@ def git(self, *args,
         if args[0] in close_panel_for:
             sublime.active_window().run_command("hide_panel", {"cancel": True})
 
+        log_stderr_for = savvy_settings.get("log_stderr_for") or []
```

Comment:
Agree that few users would object to seeing the panel and its contents earlier. I'd love to make this the new `show_panel_for` behaviour.

The stdout / stderr split is an implementation detail: To avoid introducing threading I used blocking reads. We can only consume 1 stream while blocking at a time, so the user is forced to choose.

The real solution is to use threading to read and concatenate both streams.